src/Process.js

```diff
@@ -27,7 +27,9 @@ export const createProcess = (
   // as they can only be the first command, so take out the command, and add
   // any other bits into the args
   const runtimeExecutable = workspace.pathToJest;
-  const parameters = runtimeExecutable.split(' ');
+  const parameters = runtimeExecutable.match(
+    /(?:"[^"\\]*(?:\\[\S\s][^"\\]*)*"|'[^'\\]*(?:\\[\S\s][^'\\]*)*'|(?:\\\s|\S))+/g
```

Comment:
I think I get what you are trying to do, but it is a bit complex for the mere mortal: nested non-capture-groups, [\S\s] (isn't white-space + non-white-space = well, everything?)... I was having a hard time following, wondering if it is possible to simplify it, came up with this 
```
/(\S*".*?")|(\S*'.*?')|(\S*\\\s\S+)|\S+/g
```
there are really just 3 patterns we want to capture beyond the non-white-space: single, double quotes and backslash escape... tested with the jest and the following, both seem to work:

```
"/a/b is a directory/c" --without /x/y/z useThis="I am a 'nested single quote' test" single='this is a single quote'  backslash=a\\ space/b/c 
```

however, the nested quotes cause problems for both regex patterns:
```
--nestedQuote="before \\"after this\\" end" --nestedSingleQuote='before \\'after this\\' end'
```
while this can be easily fixed with lookbehind (I am sure you can also fix your regex pattern), it does start to get more complicated and ugly... I can be talked into not to support it as I don't see how this will occur in jest command line...
```
/(\S*".*?(?<!\\)")|(\S*'.*?(?<!\\)')|(\S*\\\s\S+)|\S+/g
```

anyway, just to throw some idea out there to see if we can get it simplified a bit...

BTW, to make future maintenance easier, how about adding some comments to explain the regex patterns?

Reply:
The reason for `[\S\s]` is that it will match newlines, whereas `.` will not. This is in case of something like this

```sh
"/a/b is a directory/c" --without /x/y/z useThis="I am a 'nested 
single quote' test" single='this is a single quote'  backslash=a\\ space/b/c
```

Also, I'm pretty sure that 

```sh
--nestedQuote="before \\"after this\\" end"
```

should be two matches. Whereas

```sh
--nestedQuote="before \"after this\" end"
```

should be one match.

I noticed that the regex you suggested split

```sh
"/a/b is\" adirectory/c"
```

into two matches. It's unlikely that someone would have a quote in their path, but it is possible.

Not sure how to simplify it and still support these cases.